infra/feast-operator/api/v1alpha1/featurestore_types.go

```diff
@@ -99,19 +99,20 @@ type OfflineStorePersistence struct {
 
 // OfflineStoreFilePersistence configures the file-based persistence for the offline store service
 type OfflineStoreFilePersistence struct {
-	// +kubebuilder:validation:Enum=dask;duckdb
+	// +kubebuilder:validation:Enum=file;dask;duckdb
 	Type      string     `json:"type,omitempty"`
 	PvcConfig *PvcConfig `json:"pvc,omitempty"`
 }
 
 var ValidOfflineStoreFilePersistenceTypes = []string{
 	"dask",
 	"duckdb",
+	"file",
```

Comment:
it appears this is the same as `dask`... so maybe we don't include it.

Reply:
thoughts?

Reply:
It is supported and not deprecated yet, so I think we should include it as well

Reply:
it was removed from the doc since v0.40
https://docs.feast.dev/v0.40-branch/reference/offline-stores/overview